### ISSUE TITLE
MD5 fingerprint issue

### ISSUE THREAD

**Issue by faxe1008** [calabash/calabash-android#793]
I tried resigning the apk but it only generates this error message:

```
/app/build/outputs/apk/app-debug.apk 
/var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/helpers.rb:202:in `extract_md5_fingerprint': No MD5 fingerprint found: (RuntimeError)
 Alias name: androiddebugkey
Creation date: 2016 Nov 14
Entry type: PrivateKeyEntry
Certificate chain length: 1
Certificate[1]:
Owner: C=US, O=Android, CN=Android Debug
Issuer: C=US, O=Android, CN=Android Debug
Serial number: 1
Valid from: Mon Nov 14 11:29:16 CET 2016 until: Wed Nov 07 11:29:16 CET 2046
Certificate fingerprints:
	 SHA1: 64:9A:D7:8C:CA:2C:0F:D7:BC:7D:17:D4:28:BF:03:02:7F:07:A0:CC
	 SHA256: 00:3E:1E:2B:A7:EF:A5:73:C6:8F:5A:FE:97:C6:32:68:88:19:20:84:74:CD:1C:09:C0:BD:10:FA:A1:89:58:29
Signature algorithm name: SHA1withRSA
Subject Public Key Algorithm: RSA (1024)
Version: 1
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/java_keystore.rb:41:in `initialize'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/java_keystore.rb:116:in `new'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/java_keystore.rb:116:in `keystore_from_settings'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/java_keystore.rb:95:in `get_keystores'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/helpers.rb:157:in `sign_apk'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/helpers.rb:118:in `block in resign_apk'
	from /usr/lib/ruby/2.3.0/tmpdir.rb:89:in `mktmpdir'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/lib/calabash-android/helpers.rb:112:in `resign_apk'
	from /var/lib/gems/2.3.0/gems/calabash-android-0.8.4/bin/calabash-android:133:in `<top (required)>'
	from /usr/local/bin/calabash-android:23:in `load'
	from /usr/local/bin/calabash-android:23:in `<main>'

```
When I look up the fingerprints in the debug.keystore file I seems like I don't have the fingerprint:
` keytool -list -v -keystore debug.keystore -alias androiddebugkey -storepass android -keypass android`

```
Alias name: androiddebugkey
Creation date: Nov 14, 2016
Entry type: PrivateKeyEntry
Certificate chain length: 1
Certificate[1]:
Owner: C=US, O=Android, CN=Android Debug
Issuer: C=US, O=Android, CN=Android Debug
Serial number: 1
Valid from: Mon Nov 14 11:29:16 CET 2016 until: Wed Nov 07 11:29:16 CET 2046
Certificate fingerprints:
	 SHA1: 64:9A:D7:8C:CA:2C:0F:D7:BC:7D:17:D4:28:BF:03:02:7F:07:A0:CC
	 SHA256: 00:3E:1E:2B:A7:EF:A5:73:C6:8F:5A:FE:97:C6:32:68:88:19:20:84:74:CD:1C:09:C0:BD:10:FA:A1:89:58:29
Signature algorithm name: SHA1withRSA
Subject Public Key Algorithm: RSA (1024)
Version: 1
```
I tried creating a new debug.keystore file or using one of a colleague but it didn't work either.

**Comment by TobiasRoikjer** [comment]
Is this the same issue as this https://github.com/calabash/calabash-android/issues/752?

**Comment by faxe1008** [comment]
I tried reinstalling java-8-oracle but it didn't work either

**Comment by TobiasRoikjer** [comment]
Fixed in https://github.com/calabash/calabash-android/commit/cd9f945d9cdb7d1a305e0fbaf2c704b4d3d29656

Released in 0.9.0
